src/app/auth/login/login.component.scss

```diff
@@ -92,7 +92,9 @@
                         border: 1px solid $primary;
                     }
 
-                    
+                    &:valid {
+                        border: 1.5px solid #039855;
```

Comment:
Kindly assigned this color to a variable and make it reusable it. It  has been used at other place